pages/index.tsx

```diff
@@ -1,19 +1,29 @@
-import { useSession, signOut } from "next-auth/client";
 import Wrapper from "@components/SDCWrapper";
-
+import { WelcomeToSDC } from "@components/WelcomeToSDC";
+import { ProgramList } from "@components/ProgramList";
+import { Box, Flex, Divider, Spacer, Heading } from "@chakra-ui/react";
 export default function Component(): JSX.Element {
-    const [session, loading] = useSession();
-    if (session) {
-        return (
-            <Wrapper session={session}>
-                Signed in as {session.user.email} <br />
-                <button onClick={() => signOut()}>Sign out</button>
-            </Wrapper>
-        );
-    }
     return (
-        <Wrapper session={session}>
-            Not signed in <br />
+        <Wrapper>
```

Comment:
Should we be passing in session here to know if we should display the sign in button or the "My account" button?